epoch/api/README.md

```diff
@@ -28,7 +28,26 @@ The epoch node uses the plain HTTP protocol for its API endpoints during the tes
 
 ## WebSocket API definition
 
+### General Request/Response types
```

Comment:
Does paragraph need to be added on presence of "responses without requests" (or however we call them)? And shall the subscription/unsubscription logic be described in each of them? (I am trying to understand how subscription to documented response `mined_block` comes from.)

Reply:
> And shall the subscription/unsubscription logic be described in each of them?

I correct myself: as the the separate list of files e.g. `chain_ws_api.md` are meant to be pseudo-schemas (rather than intended usage), maybe that is not the correct place. Though this may need to be documented somewhere.

Reply:
Currently every single websocket client is subscribed to the `mine_block` events

Reply:
Maybe adding a note in ["User API - intended usage"](https://github.com/aeternity/protocol/pull/21/files#diff-8b16649bc0188ba5b35d5f92f6b8656aR52) listing all of the notifications the WS client is implicitly subscribed to would help the reader.